Content.Server/GameObjects/EntitySystems/BodySystem.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Content.Server.GameObjects.Components.Body;
+using Content.Server.GameObjects.Components.Metabolism;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects.Systems;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    [UsedImplicitly]
+    public class BodySystem : EntitySystem
+    {
+        public override void Update(float frameTime)
+        {
+            foreach (var body in ComponentManager.EntityQuery<BodyManagerComponent>())
+            {
+                body.PreMetabolism(frameTime);
+            }
+
+            foreach (var metabolism in ComponentManager.EntityQuery<MetabolismComponent>())
+            {
+                metabolism.Update(frameTime);
+            }
+
+            foreach (var body in ComponentManager.EntityQuery<BodyManagerComponent>())
+            {
+                body.PostMetabolism(frameTime);
+            }
+        }
+    }
```

Comment:
Do we need to update bodyparts/networks every frame?

Reply:
For most (read: all of them right now) no, since they use accumulatedframetime internally. Other body parts and networks might though, or they might use different delays to the ones that the current ones do (1 second).